### ISSUE TITLE
False positive : abbapps.abbotsleigh.nsw.edu.au 

### ISSUE THREAD

**Issue by hur-github** [mitchellkrogza/phishing#281]
### Domain/URL/IP(s) you believes NOT to be Phishing

```CSS
abbotsleigh.nsw.edu.au
https://abbapps.abbotsleigh.nsw.edu.au
https://abbapps.abbotsleigh.nsw.edu.au/fadcauth/login.html
```

### Whitelist type

- [ ] `1 to 1` match
- [X] `ALL` subdomains or `REGEX`
- [ ] `RZD` all sub- and top level domains matching given record(s)

### More Information

- [ ] mitchellkrogza Phishing
- [ ] mitchellkrogza Phishing.Database
- [ ] Website was hacked
- [ ] Listed as Phishing on Phishtank or OpenPhish
- [X] Listed on VirusTotal
- [ ] Other (Please tell us where in *Related external source* below)

### Related external source

```Markdown
https://www.virustotal.com/gui/url/b1d29cffaedc133aab3bf428fdce627a0a8f22a64f808ee53fd642a90baab280
https://www.virustotal.com/gui/url/268846513b97eaea5a7c685a88cedb9b419ac51a342389eee830a094bc0037db
```

### Describe the issue

1. Spirillen closed my pull request 280. and emailed me that "As the you are blocking our tests we are unable to work with this issue. Please open new issue, once you have solved your server issues".
2. Please let me know what error you get.
3. Please let me know what IP your testing is coming from.

**Comment by hur-github** [comment]
Closed pull request: Update falsepositive.list- abbapps.abbotsleigh.nsw.edu.au #280

**Comment by spirillen** [comment]
Still no proves for cleaning, what have been happening, since you have spread phishing.

I was pretty clear on this in the first PR i declined, as you (still) haven't provided any information that can help clarifying, why should be whitelisted.

Whiteout a explanation, this issue will be closed as well.

**Comment by barrettp-abbotsleigh** [comment]
@spirillen.

Just stepping in for Rick as I am the Technical Services Manager here at Abbotsleigh.

We are a K-12 girls school and we host a variety of on-premise/cloud based workloads. The domain you have listed and phishing and ultimately the url of ffadcauth/login.html is the authentication page for our on-premise reverse proxy.

Likely we are getting hit is because we also use Office 365 and we brand our login page the same as Office 365 to provide users with a consistent and seamless experience.

We would be interested to better understand why and what specific things have caused our legitimate login screen to be marked as malicious.

Please let me know what  "proof" you require ?

**Comment by spirillen** [comment]
Curious:

> K-12 girls school

What is that? I'm on the other side of our little blue planet...

> The domain you have listed

Well that's is not me, just as you , I'm stepping in for @mitchellkrogza that's why I do not have access to the logs for _why_ a url/domain might have been added, but most of the time, there was a reason for the listing.

> Likely we are getting hit because we also use Office 365 and we brand our login page the same as Office 365 to provide users with a consistent and seamless experience.

Sounds rather plausible.

> Please let me know what "proof" you require ?

Words like these... and maybe a screenshot as I do not have access to that uri

![image](https://github.com/mitchellkrogza/phishing/assets/44526987/8d0a11b8-ae09-4cba-999d-81658c5a3bd5)

**Comment by barrettp-abbotsleigh** [comment]
@spirillen 

We are a Kindergarten to Year 12 private school in Sydney, NSW Australia.

Your 403 error is likely a block on our ADCs WAF as that typically throws the `Request forbidden by administrative rules error`

Screenshot below of the authentication page.
![image](https://github.com/mitchellkrogza/phishing/assets/45025106/c8eb44ac-7f6b-443e-af8e-d4a0e085c963)

You will see that it looks almost the same as our branded Office 365 page.

![image](https://github.com/mitchellkrogza/phishing/assets/45025106/5799e314-3ad9-47de-902f-7a961de912b3)

**Comment by spirillen** [comment]
Thanks, I'll allow it for now.

**Comment by barrettp-abbotsleigh** [comment]
> Thanks, I'll allow it for now.

Thanks @spirillen , much appreciated.

**Comment by hur-github** [comment]
@spirillen in VirusTotal, the URLs are still flagged as Phishing. Please whistlist them as clean. thanks.

https://abbapps.abbotsleigh.nsw.edu.au
abbotsleigh.nsw.edu.au
https://abbapps.abbotsleigh.nsw.edu.au/fadcauth/login.html
Please refer to:
https://www.virustotal.com/gui/domain/abbapps.abbotsleigh.nsw.edu.au
https://www.virustotal.com/gui/url/268846513b97eaea5a7c685a88cedb9b419ac51a342389eee830a094bc0037db

**Comment by spirillen** [comment]
> @spirillen in VirusTotal, the URLs are still flagged as Phishing. Please whistlist them as clean. thanks.

It can take up to a couple of days from the commit to you are read the change in virus total. This is done to the frequency of builds of new lists in https://github.com/mitchellkrogza/phishing.Database and when VT is drawing the lists from @mitchellkrogza projects

So re-check on monday and let see what happens over the weekend

**Comment by hur-github** [comment]
@spirillen sorry to let you know that in VirusTotal, the URLs are still flagged as Phishing. Email was sent to Virustotal but unfortunately no reply has been received yet.

**Comment by spirillen** [comment]
> @spirillen sorry to let you know that in VirusTotal, the URLs are still flagged as Phishing. Email was sent to Virustotal but unfortunately no reply has been received yet.

Yeah VT can be rather slow on this, but have you checked, that your domain is gone from the PD lists?

**Comment by spirillen** [comment]
```bash
git grep -ri '\.nsw\.edu\.au'
active-domains.adblock:||englishtextualconcepts.nsw.edu.au^
phishing-domains-ACTIVE.txt:englishtextualconcepts.nsw.edu.au
phishing-domains/output/domains/ACTIVE/list:englishtextualconcepts.nsw.edu.au
phishing-domains/output/hosts/ACTIVE/hosts:0.0.0.0 englishtextualconcepts.nsw.edu.au
phishing-domains/output/splited/ACTIVE:englishtextualconcepts.nsw.edu.au                                                                    ACTIVE      DNSLOOKUP
phishing-links-INACTIVE.txt:http://englishtextualconcepts.nsw.edu.au/sites/default/files/a/login/1/bill%20(2).php
phishing-links/output/domains/INACTIVE/list:http://englishtextualconcepts.nsw.edu.au/sites/default/files/a/login/1/bill%20(2).php
phishing-links/output/splited/INACTIVE:http://englishtextualconcepts.nsw.edu.au/sites/default/files/a/login/1/bill%20(2).php                INACTIVE    URL
```

**Comment by hur-github** [comment]
@spirillen I had the reply from VirusTotal. Phishing.Database still flagged Abbotsleigh School (https://abbapps.abbotsleigh.nsw.edu.au)as Phishing in VT. Please update it as clean or contact VT for more information.

Please refer to below for the reply from VT:
 
Ana Tinoco (VirusTotal) 
Oct 26, 2023, 9:33 AM GMT+2 
Hello,
 
We don't scan files or URLs ourselves. We receive results from different AV vendors. If it's still listed as malware in our results it's because the AV vendor hasn't sent us updated results. Once they send us updated results, the information is automatically updated in VirusTotal.
 
Please contact the AV vendor and let them know that they have to send updated results to VirusTotal.
 
Thank you
 
Best regards,
Ana Tinoco - VirusTotal - www.virustotal.com

**Comment by spirillen** [comment]
>  the information is automatically updated in VirusTotal.

This is the piece of information I refereed to, and the part where I, as @spirillen, have no power, the same comes to the build of the phishing list that is generated in /phishing.database only @mitchellkrogza and @funilrys have powers their on the backend and lists building.

I can, as me, can't help you any further for now, the situation requires @mitchellkrogza and @funilrys as I did not find your domain on the phishing database https://github.com/mitchellkrogza/phishing/issues/281#issuecomment-1779953845

**Comment by hur-github** [comment]
@mitchellkrogza @funilrys 
the URLs are still flagged as Phishing. Please whistlist them as clean. thanks.

https://abbapps.abbotsleigh.nsw.edu.au/
abbotsleigh.nsw.edu.au
https://abbapps.abbotsleigh.nsw.edu.au/fadcauth/login.html
Please refer to:
https://www.virustotal.com/gui/domain/abbapps.abbotsleigh.nsw.edu.au
https://www.virustotal.com/gui/url/268846513b97eaea5a7c685a88cedb9b419ac51a342389eee830a094bc0037db

[Abbotsleigh Sign In](https://abbapps.abbotsleigh.nsw.edu.au/)
[Abbotsleigh Sign In](https://abbapps.abbotsleigh.nsw.edu.au/fadcauth/login.html)
[VirusTotal](https://www.virustotal.com/gui/domain/abbapps.abbotsleigh.nsw.edu.au)
VirusTotal
[VirusTotal](https://www.virustotal.com/gui/url/268846513b97eaea5a7c685a88cedb9b419ac51a342389eee830a094bc0037db)
VirusTotal

refer to below:
This is the piece of information I refereed to, and the part where I, as @spirillen, have no power, the same comes to the build of the phishing list that is generated in /phishing.database only @mitchellkrogza and @funilrys have powers their on the backend and lists building.

I can, as me, can't help you any further for now, the situation requires @mitchellkrogza and @funilrys as I did not find your domain on the phishing database https://github.com/mitchellkrogza/phishing/issues/281#issuecomment-1779953845
